GitUI/CommandsDialogs/SettingsDialog/Pages/RevisionLinksSettingsPage.resx

```diff
@@ -117,10 +117,68 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <metadata name="toolStripManageCategories.TrayLocation" type="System.Drawing.Point, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
+    <value>17, 3</value>
+  </metadata>
+  <assembly alias="System.Drawing" name="System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
+  <data name="AddGitHubTemplatesMenuItem.Image" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
+    <value>
+        iVBORw0KGgoAAAANSUhEUgAAACAAAAAgCAYAAABzenr0AAAABGdBTUEAALGPC/xhBQAAA1xJREFUWEe1
+        V0trU0EYjW8XvmXm3jwaY00NGNBFcB8EETcuhG5EQV3oxp3FjSAxP0B8BVfuxT+guyq4qIKoXQhFEB9Q
+        F1o1rUKLtfGc6XdjcvMlvebGA4ck3+N8d76ZuTNJREWxWFyftPaIb23NN2YcnPKsnSP53dngYwxjJS0+
+        csb4EL7jGVMHGxFZZw5zRebfkc/nN0CkCv5QCkQic6lBLZGNBhn1hCbaD6kVuRsZa/djPj9qQnFITWpL
+        GR1u5P+heEBqd+2EzHlb25HwGrwH+5dWe0R+Q+598HmrnTXUNQFHtTWQTHreOfrK5fJaz/POYst9cr7l
+        7TcJ8cfgI3x/Bft3+qAzA9uFYCtiWx4P9AKyFn1NyKLrWO2+7x+UEIeiMZtg21epVFaLqYlGo7EqnU7v
+        zWaz28XkgAff3aGLWm1TAQP3eVsQmUwmSxLSN1KpVFbTZk0XwFbBoL5k0L5TLigG0LGjmjZYd9PEV6fi
+        dG1CB3aJTt/AtOyE1metBmuz/TXNCfsl0YgNLMozXWrU6BzXnJlMZkTyYyOXy23TarA2H2Cqw4H2c1VL
+        /kAAzfcddVA74fZ02GHtW8kbGFDsWbgOa/PJfnY4jJmVvIGBgwrXYW21NY6eZyU3NrAGNqLOfLgGbO/4
+        AE/DDhJb5KTkx0aPrT6RSBpzQ3Vizga1EFHogVaDtdXDIiASx0Sjb2AgpzVtkrUThUJhM37Mhp3CJTxE
+        tVQqrRO9yOCBxQFAYzGkGXCWh5sLRuDtpsPaaynPOyFHrLNhFB8QU0lZexgvqB0uScHw8PBWLN5DOMYv
+        I/9NU1Oh73m3JA3XMGPyKLDgHMZ8ZSE+HURehBNhe6IdxwRy1bdqmKg1P2TtHklbBoxXW4LqOEQw2MwI
+        7G0HCbpzTFI6gJOv3BrbjeympPwFr0mhtt+lXS4rF2G7zhtSrys2pyfI70rU6KqBy8MQgqYleJFnubgi
+        A3m/JF/jNGtIqI60tQcYKAmLGP1DcAxb5jw+r4yOjq6RUBXMkdwwp6ktYb3B+UerXioijZX+9yHmdzgH
+        WpO8mklINMg1vQK2vcP5XpcQFYhZCmKRuwBWV8rpCW4X7NmbEKxjJHO8ootLBQrOMA6ftUFeatyVPHzl
+        1sDpQ/e2yM8VkEj8AXnsU9Jt20W4AAAAAElFTkSuQmCC
+</value>
```

Comment:
It would be a good idea to use reference actual images than a serialised representation of them.
Images referenced from a file are easy to replace, when they are serialised - not so much.